onenote/.gitignore

```diff
@@ -0,0 +1,45 @@
+# Compiled class file
```

Comment:
What's the purpose for two .gitignore files?. We can use root .gitignore

Reply:
Yes. Agreed.
Done.